router/src/main/resources/envoy.yaml.template

```
@@ -42,9 +42,9 @@ static_resources:
       connect_timeout: 1s
       upstream_connection_options:
         tcp_keepalive:
-          keepalive_probes: 3
-          keepalive_time: 300
-          keepalive_interval: 30
+          keepalive_probes: '${KEEPALIVE_PROBES}'
```

Shall we prefix ADAPTER to make this config more specific?
```suggestion
          keepalive_probes: '${ADAPTER_KEEPALIVE_PROBES}'
```

---

I guess we can keep this as it is and use it in all places. Hence +1